langchain4j-azure-open-ai/src/main/java/dev/langchain4j/model/azure/AzureOpenAiStreamingChatModel.java

```diff
@@ -343,6 +398,8 @@ public static class Builder {
         private ProxyOptions proxyOptions;
         private boolean logRequestsAndResponses;
         private OpenAIClient openAIClient;
+        private OpenAIAsyncClient openAIAsyncClient;
+        private boolean useAsyncClient = false;
```

Comment:
I would make it `true` by default, WDYT?

Reply:
done. Makes totally sense, syncClient is actually obsolete for StreamingChatModel.